config/tools/generate_configd.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 MIT Digital Currency Initiative,
+//                    Federal Reserve Bank of Boston
+//                    MITRE Corporation
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "config_generator.hpp"
+
+#include <iostream>
+#include <random>
+
+auto main(int argc, char** argv) -> int {
```

Comment:
This doesn't seem to be a daemon; what does the `d` in the filename refer to?

Reply:
Misunderstood the convention. Removing the d.

DONE